src/Symfony/Component/DependencyInjection/ParameterBag/ParameterBag.php

```diff
@@ -264,6 +219,9 @@ public function escapeValue($value)
         return $value;
     }
 
+    /**
+     * {@inheritdoc}
+     */
     public function unescapeValue($value)
```

Comment:
this looks like an issue with resolveValue, isn't it? I'd expect resolveValue to resolve parameters and escaping in the same run